actionview/lib/action_view/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Engine # :nodoc:
 
     initializer "action_view.per_request_digest_cache" do |app|
       ActiveSupport.on_load(:action_view) do
-        if app.config.consider_all_requests_local
+        if app.config.consider_all_requests_local && !Rails.env.test?
           app.executor.to_run ActionView::Digestor::PerExecutionDigestCacheExpiry
```

Comment:
Don't you think it's overkill? I mean, the only place where it's used is dev env. I can't think any other scenario up.